/airbot_lidar_merger:
  ros__parameters:
    flip1: false
    flip2: false
    inverse1: false
    inverse2: false
    laser1Alpha: 180.0
    laser1AngleMax: 270.0  # 조정된 각도 범위
    laser1AngleMin: 90.0  # 조정된 각도 범위
    laser1B: 0
    laser1G: 0
    laser1R: 255
    laser1XOff: 0.15  # 앞쪽 라이다의 X 오프셋 (앞으로 0.05m)
    laser1YOff: 0.0
    laser1ZOff: 0.0
    laser2Alpha: 0.0
    laser2AngleMax: 270.0  # 조정된 각도 범위
    laser2AngleMin: 90.0  # 조정된 각도 범위
    laser2B: 255
    laser2G: 0
    laser2R: 0
    laser2XOff: -0.15  # 뒤쪽 라이다의 X 오프셋 (뒤로 0.05m)
    laser2YOff: 0.0
    laser2ZOff: 0.0
    pointCloudTopic: cloud_in
    pointCloutFrameId: base_scan
    qos_overrides./parameter_events.publisher.depth: 1000
    qos_overrides./parameter_events.publisher.reliability: best_effort #best_effort
    scanTopic1: /scan_front
    scanTopic2: /scan_back
    show1: true
    show2: true
    use_sim_time: false #false

/pointcloud_to_laserscan:
  ros__parameters:
    angle_increment: 0.0157 #2.68/400 = 0.0157 #이전 값 : 0.0087
    angle_max: 3.14
    angle_min: -3.14
    inf_epsilon: 1.0
    max_height: 1.0
    min_height: 0.0
    qos_overrides:
      /parameter_events:
        publisher:
          depth: 1000
          reliability: best_effort
    queue_size: 10000
    range_max: 8.0
    range_min: 0.1
    scan_time: 0.1
    target_frame: base_scan
    transform_tolerance: 0.01
    use_inf: true
    use_sim_time: false #false
